DPGAnalysis/Skims/python/HighMETSkim_cff.py

```diff
@@ -28,31 +28,74 @@
     minNumber = cms.uint32(1),
     )
 
-pfMETSelSeq = cms.Sequence(pvFilter*
-                           HBHENoiseFilter*
+pfMETSelSeq = cms.Sequence(
+			   pvFilter*
+                           ##HBHENoiseFilter*
```

Comment:
Here and in a few more places you have removed the HBHENoiseFilter.
This will change the behavior of the execution sequence.
Is this intentional?

The old behavior would be preserved by replacing HBHENoiseFilter with HBHENoiseFilterResultProducer*HBHENoiseFilter as done in other places.


Please remove the commented out part if it's here just as a reminder that it was here in the past and is not supposed to be used.